src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
+      transition: 300ms;
```

Comment:
1) move transition to static state instead of hover one
2) add transition-property (-ies) to show what styles you exactly change
3) create a variable for transition-duration in case you've already used this value twice